RevenueCatUI/CustomerCenter/Views/WrongPlatformView.swift

```diff
@@ -178,8 +178,10 @@ struct WrongPlatformView_Previews: PreviewProvider {
     }
 
     private static func getPurchaseInformation(for customerInfo: CustomerInfo) -> PurchaseInformation {
-        return PurchaseInformation(entitlement: customerInfo.entitlements.active.first!.value,
-                                   transaction: customerInfo.subscriptionsByProductIdentifier.values.first!)
+        return PurchaseInformation(
+            entitlement: customerInfo.entitlements.active.first!.value,
+            transaction: customerInfo.subscriptionsByProductIdentifier.values.first!,
+            customerInfoRequestedDate: Date())
```

Comment:
any reason why this is not getting the request date of the customer info?

Reply:
oh this is in the previews. sorry 😮‍💨 

nevermind then!

Reply:
https://github.com/RevenueCat/purchases-ios/pull/4805/files#diff-d35a1e787947e08d6465b6c369913eea0f5cae5f1b29428539e35162a4092ec3R184